app/boards/shields/reviung5/reviung5.keymap

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+#include <dt-bindings/zmk/outputs.h>
+
+#define BASE 0
+#define BLE 1
+
+&encoder { status = "okay"; };
```

Comment:
No need to do this for the keymap if there's just one encoder, just enable it in the overlay where the encoder node is defined.

Reply:
Hopefully it's okay now.